torch/csrc/utils/device_lazy_init.cpp

```diff
@@ -27,6 +27,13 @@ void device_lazy_init(at::DeviceType device_type) {
     throw python_error();
   }
 
+  if (device_type == at::DeviceType::PrivateUse1) {
```

Comment:
@albanD  our device need to `_lazy_init()`， a  `._lazy_init()` method is defined on our device module. 
this modification can handle both define _lazy_init() and not define _lazy_init().
please review it, thanks.

Reply:
please review my PR， thanks. @ezyang @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler @amjames @desertfire @chauhang